this_is_a_test.py

```diff
@@ -1,2 +1,3 @@
 # My passwords
 a_password = "my_sp3c1f1c_pa$$w0rd_trav1s_ll0yd_1"
+fp_testing_pwd = 'bht4zgb7gfp@ftz_ZBH'
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- c6978ccd-ed01-46df-801f-50e3e71c878f -->

Reply:
#cycode_secret_ignore_everywhere

Reply:
:+1: Cycode has finished processing your request. Please review your status checks.